config/prod/prow/config_knative.yaml

```diff
@@ -134,7 +134,8 @@ presubmits:
         memory: 16Gi
   - custom-test: contour-latest
     dot-dev: true
-    always_run: false
+    always-run: false
+    run-if-changed: "^third_party/contour-latest/*"
```

Comment:
Cool!

@tcnghia @nak3 we should consider something similar for the istio directories and the mesh legs?

@markusthoemmes @davidor @jmprusi same for kourier?

Reply:
Yes, definitely we would like to have it.

cc @JRBANCEL as trying to change the directory structure.